test_scripts/Resumption/Handling_errors_from_HMI/013_resume_one_rpc_late_response_unexpected_disconnect.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. <Rpc_n> related to resumption is sent by app
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App re-registers with actual HashId
+-- SDL does:
+--  - start resumption process
+--  - send <Rpc_n> request to HMI
+-- 4. HMI responds with SUCCESS resultCode to <Rpc_n> request when default timeout (10s) expires
```

Comment:
In this case doesnt this mean the HMI will be displaying resumed data that Core thinks has failed?